.github/workflows/ci.yml

```diff
@@ -56,6 +57,10 @@ jobs:
             python: 3.6
             toxdir: cli
             toxenv: cfn-format-check,cfn-lint,cfn-tests
+          - name: Integration Tests Config Checks
+            python: 3.6
+            toxdir: tests/integration-tests
+            toxenv: validate-test-configs,generate-test-config
```

Comment:
I think we only need `validate-test-configs`

Reply:
fixed